app/validators/date-range.js

```diff
@@ -0,0 +1,7 @@
+import moment from 'moment'
+
+export default function validateDateRange(options) {
+  return (key, newValue) => {
+    return moment().range(options.min, options.max).contains(moment(newValue), false)
```

Comment:
So would you wrap the `date-range` validator in something like `contribution-date` validator, that has access to the model?